src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -46,5 +47,17 @@ public override string ToString()
                 return $"{Level}: {Message}";
             }
         }
+
+        static readonly AsyncLocal<ScenarioContext> asyncContext = new AsyncLocal<ScenarioContext>();
```

Comment:
From a formatting standpoint, this declaration should probably be above the `LogItem` declaration.